src/app/lib/config/services/hausa.ts

```diff
@@ -97,7 +97,7 @@ export const service: DefaultServiceConfig = {
         videoPlayer: 'Video player',
       },
       liveExperiencePage: {
-        liveLabel: 'Kai-tsaye',
+        liveLabel: 'Kai tsaye',
         liveCoverage: 'Rahoto kai-tsaye',
```

Comment:
Shall we check with editorial about this & whether it also needs the hypen removed?

```suggestion
        liveCoverage: 'Rahoto kai tsaye',
```

Reply:
And there's another one on line 53

Reply:
Checking with editorial on this, but will merge this for now and follow up later if required.